web/packages/shared/utils/highbar.ts

```diff
@@ -333,167 +329,3 @@ export function debounce<T extends (...args: any) => any>(
   debounced.flush = flush;
   return debounced;
 }
-
-interface MapCacheType {
```

Comment:
I decided to get rid of `memoize` because it had many `this` issues.
It was only used in one place, which was trivial to refactor, and this old-style code was hard to maintain (we already had a few `any` and TS/ESLint ignores there).